src/main/resources/io/jenkins/plugins/pipeline_elasticsearch_logs/ElasticSearchConfiguration/config.jelly

```diff
@@ -22,9 +22,9 @@
     </f:entry>
     <f:entry >
       <f:dropdownDescriptorSelector title="RunID Provider" field="runIdProvider"/>
-    </f:entry>  
+    </f:entry>
     <f:entry field="connectionTimeoutMillis" title="Connection Timeout (ms)">
-      <f:textbox/>
+      <f:number clazz="number" min="0" max="2147483647" default="10000"/>
```

Comment:
We have the default hard coded here and in the config class. Maybe we should provide a getDefaultConnectionTimeout method, so in case we change the default, we don't have to adjust 2 places.

Reply:
Before wasting time for research or try&error, do you know how to do it? I guess a `getDefault...` in the Descriptor? Will it automatically work or do we need to use something special in `default=""`?

Reply:
We can do this later. I'll merge

Reply:
yes we need to call the method in the jelly. There is not automatic filling of defaults